JASP-Engine/JASP/R/mlRegressionRegularized.R

```diff
@@ -311,10 +311,10 @@ mlRegressionRegularized <- function(jaspResults, dataset, options, ...) {
         ggplot2::geom_ribbon(data = regressionResult[["cvMSELambda"]], mapping = ggplot2::aes(ymin = MSE - sd, ymax = MSE + sd), fill = "grey90") +
         JASPgraphs::geom_line() +
         ggplot2::scale_x_continuous("\u03BB", breaks = xBreaks, labels = xBreaks) +
-        ggplot2::scale_y_continuous("Cross-Validated \nMean Squared Error", breaks = yBreaks, labels = yBreaks) +
+        ggplot2::scale_y_continuous(gettextf("Cross-Validated %sMean Squared Error", "\n"), breaks = yBreaks, labels = yBreaks) +
```

Comment:
@FransMeerhoff we really need some standardized way to deal with newlines. I don't think this is what was done in other analyses?